### PR TITLE
Flow

### PR DESCRIPTION
**Pull request by jfrazier-eth**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "@hapi/vision": "^6.1.0",
     "@opensea/stream-js": "^0.0.21-rc.1",
     "@poprank/rankings": "^1.1.20",
-    "@reservoir0x/sdk": "0.0.301",
+    "@reservoir0x/sdk": "^0.0.302",
```
Comment (georgeroman) [inline package.json:37]: Let's keep a fixed version.

File: src/api/endpoints/execute/get-execute-bid/v4.ts [hunk 2]
```diff
@@ -614,7 +627,7 @@ export const getExecuteBidV4Options: RouteOptions = {
             ) {
               approvalTx = currency.approveTransaction(
                 maker,
-                Sdk.Forward.Addresses.Exchange[config.chainId]
+                Sdk.Infinity.Addresses.Exchange[config.chainId]
```
Comment (georgeroman) [inline src/api/endpoints/execute/get-execute-bid/v4.ts:630]: This shouldn't get changed.

**Review by georgeroman — CHANGES_REQUESTED** [review]

File: src/api/endpoints/execute/get-execute-bid/v4.ts (hunk 2 above)
Comment (jfrazier-eth) [reply src/api/endpoints/execute/get-execute-bid/v4.ts:630]: As in I should open another PR for this?

Comment (georgeroman) [reply src/api/endpoints/execute/get-execute-bid/v4.ts:630]: My bad, I though this was some other part of the code.

**Pull request merged**